### ISSUE TITLE
Do not display complete error messages when test fails

### ISSUE THREAD

**Issue by GoogleCodeExporter** [manojjsm/js-test-driver#270]
```
What steps will reproduce the problem?
1. Just run a test that fails
2.
3.

What is the expected output? What do you see instead?
I expect to see AssertError: expected false but was true.
I see just AssertError: 
There is no error details

What version of the product are you using? On what operating system?

JsTestDrive-1.3.2
Windows 7 Professional Edition 64bit
jdk-6u27

Please provide any additional information below.

It was working perfectly before the last Windows update. I guess it could be 
something related with it.
```

Original issue reported on code.google.com by `pl.todo...@gmail.com` on 23 Sep 2011 at 12:57

**Comment by GoogleCodeExporter** [comment]
```
Ok it seems the problem is browser related.

Here is brief statistics:

Chrome version 14.0.835.186 m -> messages are NOT displayed
Firefox version 6.0.2 -> messages are NOT displayed
IE version 9.0.8112.16421 -> messages are displayed
```

Original comment by `pl.todo...@gmail.com` on 23 Sep 2011 at 1:17

**Comment by GoogleCodeExporter** [comment]
```
Seen on both jstestdriver-1.3.2 and jstestdriver-1.2.2.
Exhibits on jstestdriver-1.2.2 as no text in error output at all.

Mac OSX 10.7.1
jdk 1.6.0_26
Chrome version 14.0.835.186 (no error after failed:)
Firefox 6.0.2 (output includes line numbers for affected files, but no error 
message)

Windows 7 Professional Edition 64 bit.
1.6.0_24
Chrome version 14.0.835.186

```

Original comment by `joe.math...@gmail.com` on 29 Sep 2011 at 11:20

**Comment by GoogleCodeExporter** [comment]
```
Issue 265 has been merged into this issue.
```

Original comment by `corbinrs...@gmail.com` on 29 Sep 2011 at 3:11

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.3.3a, as of r1038
```

Original comment by `corbinrs...@gmail.com` on 29 Sep 2011 at 3:11

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.3.3a, as of r1038
```

Original comment by `corbinrs...@gmail.com` on 29 Sep 2011 at 3:12
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Included back port patch for 1.2.2
```

Original comment by `joe.math...@gmail.com` on 3 Oct 2011 at 2:35

Attachments:
- [TestRunnerPlugin.js.diff](https://storage.googleapis.com/google-code-attachments/js-test-driver/issue-270/comment-6/TestRunnerPlugin.js.diff)
